Fix version. A semantic version number must be composed of exactly three
period-separated integers, and in addition it must match the version number
of openbabel. Because the version was invalid, the code in setup.py that
should look for openbabel using pkg-config didn't run and it instead assumed
it was in /usr/local, which it isn't, so the build failed.
https://github.com/openbabel/openbabel/issues/2408
--- openbabel/__init__.py.orig	2020-05-23 04:32:28.000000000 -0500
+++ openbabel/__init__.py	2022-04-12 17:24:44.000000000 -0500
@@ -2,7 +2,7 @@
 import warnings
 from . import openbabel
 
-__version__ = "3.1.1.1"
+__version__ = "3.1.1"
 
 _thismodule = sys.modules[__name__]
 
